pyomo/opt/tests/base/test_ampl.py

```diff
@@ -52,7 +52,7 @@ def test3_write_nl(self):
         """ Convert from MOD+DAT to NL """
         try:
             self.model.write(currdir+'test3.nl')
-        except pyutilib.common.ApplicationError:
+        except ApplicationError:
```

Comment:
I am confused as to why we have these `except:` clauses.

Reply:
I'm not sure either. What do you think is a better approach?